prow/pluginhelp/pluginhelp.go

```diff
@@ -30,8 +30,10 @@ type PluginHelp struct {
 	// This field may include HTML.
 	WhoCanUse string
 	// Usage is a usage string for the plugin. Leave empty if not applicable.
+	// TODO(qhuynh96): this field is being removed when plugins adapt to new struct.
```

Comment:
Checking in `TODO` comments is usually not necessary.

Reply:
Done

Reply:
> Checking in TODO comments is usually not necessary.

Why not? We should add a comment or open an issue if we are deprecating these fields.